### PR TITLE
[TASK]Setup tailwindcss

### PR DESCRIPTION
**Pull request by Diezagu**
I recently setup tailwindcss :) please don't be aggressive with your comments. Peace!

### CODE REVIEW

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
Let's add it as a js dependency instead.
This gem uses `tailwindcss 2.2.15`, so we better use latest stable version(2.2.16)

**Comment by itsmeurbi** [comment]
You can follow Tailwind official setup guide: https://tailwindcss.com/docs/installation
You will probably experiences a [PostCSS 8 issue compatibility](https://tailwindcss.com/docs/installation#post-css-7-compatibility-build), let's try updating to v8 first

**Review by bryanmg — APPROVED** [review]
LGTM!

**Comment by Diezagu** [comment]
I have already changed the setup of tailwindcss as a js dependency :)

File: Gemfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ gem 'jbuilder', '~> 2.7'
 # gem 'bcrypt', '~> 3.1.7'
 gem 'rubocop', require: false
 
+gem "tailwindcss-rails", "~> 0.5.1"
```
Comment (itsmeurbi) [inline Gemfile]: If tailwind was added as a js package, then there's no need for a gem

**Review by itsmeurbi — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -5,12 +5,16 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "inter-font", "data-turbo-track": "reload" %>
```
Comment (itsmeurbi) [inline app/views/layouts/application.html.erb:8]: Why do we need this?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -5,12 +5,16 @@
...
+    <%= stylesheet_link_tag "tailwind", "data-turbo-track": "reload" %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <%= yield %>
+    <main class="container mx-auto mt-28 px-5 flex">
```
Comment (itsmeurbi) [inline app/views/layouts/application.html.erb:16]: No needed at the moment. We haven't discuss anything about designs

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
I don't thinks this is the right way t do it.
Screen Shot 2021-12-13 at 10 18 28

**Comment by itsmeurbi** [comment]
@Diezagu since we got rid of node dependencies, we need to add Tailwind in a different way. I just created [another PR](https://github.com/itsmeurbi/medical_control_rails/pull/12) doing that.

**Comment by itsmeurbi** [comment]
Closing this PR due to comment [here](https://github.com/itsmeurbi/medical_control_rails/pull/4#issuecomment-1001782169)

**Pull request closed without merging**